### ISSUE TITLE
Gradient accumulation weighting

### ISSUE THREAD

**Issue by vadimkantorov** [NVlabs/wetectron#70]
From what I can see, in wetectron code there is no division of loss by iter_size. Is it correct? It seems that gradient magnitude will not be equivalent to a single larger batch. 

DRN does this division: https://github.com/shenyunhang/DRN-WSOD-pytorch/blob/DRN-WSOD/projects/WSL/tools/train_net.py#L103

**Comment by bradezard131** [comment]
Hey @vadimkantorov I might take this since I added it. When I ran it I originally did do the divide by iter_size, but found I was not able to reproduce the original results running 1 GPU and iter_size=8. When I removed the divide by iter_size I was able to reproduce the results. Others have had trouble with it though, it hasn't worked as well as expected as can be seen in the issues.
